frontend/src/components/payment/transactions/TransactionList.tsx

```diff
@@ -0,0 +1,99 @@
+import { useTranslation } from 'next-i18next'
+import Link from 'next/link'
+import { FunctionComponent, useState } from 'react'
+import { Transaction } from '../../../types/Payment'
+import Button from '../../Button'
+import TransactionCancelButton from './TransactionCancelButton'
+import styles from './TransactionList.module.scss'
+
+interface ListProps {
+  transactions: Transaction[]
+}
+
+const TransactionList: FunctionComponent<ListProps> = ({ transactions }) => {
+  const { t } = useTranslation()
+
+  return (
+    <table className={styles.transactionList}>
+      <thead>
+        <tr>
+          <td>{t('created')}</td>
+          <td>{t('updated')}</td>
+          <td>{t('id')}</td>
```

Comment:
I would omit this here, should be enough, if it's in the details. I also don't like to translate this :)